uaclient/entitlements/__init__.py

```diff
@@ -13,6 +13,8 @@ def cast(_, x):  # type: ignore
         return x
 
 
+ENTITLEMENT_ALIASES = {"esm": "esm-infra"}
```

Comment:
This doesn't appear to be used anywhere. (Presumably related to my above question about aliases?)

Reply:
Also vestigial and now dropped. thanks.